Sources/SwiftDocC/SwiftDocC.docc/SwiftDocC/LinkResolution.md

```diff
@@ -0,0 +1,59 @@
+# Linking between documentation
+
+Connect documentation pages with documentation links.
+
+## Overview
+
+DocC uses links to connect documentation pages to each other. 
+
+Some links are automatic or created by tools. For example symbol declarations in symbol graph files use `typeIdentifier` tokens with `preciseIdentifier` values to link to other types that are referenced in the declaration and symbol graph relationships are used to construct a default documentation hierarchy. These links use unique and exact identifiers to reference other symbols. 
+
+All other documentation links are written in the documentation content; in symbol documentation comments or in markdown or tutorial files. These "authored" documentation links are used to associate documentation extension files with symbols, form links in the documentation content itself, customize the documentation hierarchy, and organize tutorials. 
+
+Developers can write two types of documentation links:
+ - Symbol links; a symbol page surrounded by two grave accents on each side: ``MyClass/myProperty``)
```

Comment:
```suggestion
 - Symbol links; a symbol page surrounded by two grave accents on each side: ``MyClass/myProperty``
```

Reply:
I'll also wrap this in an inline code block for two reasons: to avoid a warning about an unresolved link and sinceis meant to show the authoring syntax.